src/storage/hummock_sdk/src/slice_transform.rs

```diff
@@ -215,10 +219,94 @@ impl SliceTransform for MultiSliceTransform {
     }
 }
 
+#[derive(Default)]
+pub struct SliceTransformManagerInner {
+    table_id_to_slice_transform: RwLock<HashMap<u32, SliceTransformImpl>>,
+    version: AtomicU64,
+    notify: Notify,
+}
+
+impl SliceTransformManagerInner {
+    fn update(&self, table_id: u32, slice_transform: SliceTransformImpl) {
+        self.table_id_to_slice_transform
+            .write()
+            .insert(table_id, slice_transform);
+    }
+
+    fn remove(&self, table_id: u32) {
+        self.table_id_to_slice_transform.write().remove(&table_id);
+    }
+
+    async fn acquire(&self, mut remaining_table_id_set: HashSet<u32>) -> MultiSliceTransform {
```

Comment:
Please add some docs to explain when and how this method will be used?
minor: rename `remaining_table_id_set` to `table_id_set` since the caller doesn't understand what "remaining" means.